### ISSUE TITLE
Document that GITHUB_TOKEN cannot add/edit workflows

### ISSUE THREAD

**Issue by Konard** [irgolic/AutoPR#77]
[![AutoPR Failure](https://img.shields.io/badge/AutoPR-failure-red)](https://github.com/Konard/AutoPR-tests/actions/runs/4886240680)

AutoPR encountered an error while trying to fix https://github.com/Konard/AutoPR-tests/issues/5.

## Traceback

```
Traceback (most recent call last):
  File "/app/autopr/agents/brain_agent/base.py", line 75, in generate_pr
    self._generate_pr(event)
  File "/app/autopr/agents/brain_agent/simple_v1.py", line 42, in _generate_pr
    self.commit_service.commit(current_commit, push=True)
  File "/app/autopr/services/commit_service.py", line 78, in commit
    self.repo.git.execute(["git", "push", "-f", "origin", self.branch_name])
  File "/venv/lib/python3.9/site-packages/git/cmd.py", line 1109, in execute
    raise GitCommandError(redacted_command, status, stderr_value, stdout_value)
git.exc.GitCommandError: Cmd('git') failed due to: exit code(1)
  cmdline: git push -f origin autopr/5
  stderr: 'To https://github.com/Konard/AutoPR-tests
 ! [remote rejected] autopr/5 -> autopr/5 (refusing to allow a GitHub App to create or update workflow `.github/workflows/run_hello_world.yml` without `workflows` permission)
error: failed to push some refs to 'https://github.com/Konard/AutoPR-tests''

```

**Comment by irgolic** [comment]
Hey, thanks for running AutoPR c:

A github actions's `GITHUB_TOKEN` can't edit workflows in the repo. This is a design decision by GitHub, the way to work around it is to use a personal access token instead. I'm considering making a GitHub app at some point, to get around this.

This should definitely be better documented, preferably the error should be handled by AutoPR to give a more informative message in the PR.

**Comment by irgolic** [comment]
Fixed by #79
